velox/type/Timestamp.h

```diff
@@ -160,11 +160,11 @@ struct Timestamp {
     // T - literal T
     // %T - equivalent to "%H:%M:%S" (the ISO 8601 time format)
     // so this return time in the format
-    // %Y-%m-%dT%H:%M:%S.nnnnnnnnn
+    // %Y-%m-%dT%H:%M:%S.nnn
```

Comment:
In general, it is not a good practice to rely on the output of 'toString()' method. The change here makes it so that Timestamp::toString() no longer shows the complete value as it truncates nanoseconds.

Consider adding a parameter to control the precision and have the default be 'nanoseconds'. Then, use milliseconds precision in CAST.